server/events/project_command_builder.go

```diff
@@ -114,6 +114,12 @@ func (p *DefaultProjectCommandBuilder) buildPlanAllCommands(ctx *CommandContext,
 		return nil, err
 	}
 
+	// Check for use of .yml instead of .yaml for config file.
+	hasInvalidFileExtension, _ := p.ParserValidator.HasInvalidFileExtension(repoDir)
+	if hasInvalidFileExtension {
+		ctx.Log.Warn("found %q as config file; rename using the .yaml extension %q", yaml.InvalidAtlantisYAMLFileExtension, yaml.AtlantisYAMLFilename)
+	}
+
 	// Parse config file if it exists.
 	hasRepoCfg, err := p.ParserValidator.HasRepoCfg(repoDir)
```

Comment:
Let's put the functionality inside the `HasRepoCfg` function. And change it so that if there's an `atlantis.yml` it errors out.

Reply:
Cool, moved the functionality into `HasRepoCfg` and added the error functionality.